### DISCUSSION TITLE
Object reference not set to an instance of an object.

### DISCUSSION THREAD

**Discussion by Sand216** [nptr/msstyleEditor#147] [Q&A] [answered]
Hello! I am new to msstyles, and I am making a theme for fun. I tried to test my msstyles after making some changes and got this error: 

![image](https://github.com/nptr/msstyleEditor/assets/164842290/29e95a2f-a9b6-43f5-8b13-b08d32efa9d2)

What causes this error?

As there are hundreds of objects to look through, I would like to know if there is an easier way to find the cause of this error besides going through every object.

**Comment by nptr** [comment]
No, thats a bug. The message refers to some data objects in the program itself not existing anymore.
If you can reliably reproduce/trigger this bug. please open an issue with a description of how I can reproduce it myself.

**Reply by Sand216** [reply, marked as answer]
I solved this a while ago, but I forgot to close this and give a solution.

I’ll open an issue later.

**Reply by Sand216** [reply]
I cannot figure out a way to reproduce this issue so it may have been an outlier. If I remember correctly, it was caused by setting the `IMAGEFILE` property of a part to a large number like 9999. Since this isn't reproducible and the functionality is niche it's probably okay to leave it alone.
